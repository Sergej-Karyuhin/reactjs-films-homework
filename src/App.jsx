import React from 'react';
import MovieDetails from './pages/MovieDetails';

import appStyles from './styles.scss';

const App = () => (
  <div className={appStyles}>
    <MovieDetails />
  </div>
);

export default App;
