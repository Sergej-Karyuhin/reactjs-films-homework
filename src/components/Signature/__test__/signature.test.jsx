import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import App from '../../../App';

test('Application is rendered', () => {
  const renderer = new ShallowRenderer();
  renderer.render(<App />);
  const result = renderer.getRenderOutput();
  expect(result).toMatchSnapshot();
});
