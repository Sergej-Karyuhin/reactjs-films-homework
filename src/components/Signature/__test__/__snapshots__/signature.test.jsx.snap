// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Application is rendered 1`] = `
<div>
  <Signature
    name="Serge"
  />
</div>
`;
